src/platform/plugins/shared/dashboard/tsconfig.json

```diff
@@ -54,7 +54,6 @@
     "@kbn/core-saved-objects-server",
```

Comment:
Changes to this file should be able to be reverted, since no types from vis and event-annotation should leak into dashboard